src/api/dependency-discovery/src/server.js

```diff
@@ -0,0 +1,5 @@
+const service = require('.');
+
+const port = parseInt(process.env.DEPENDENCY_GRAPH_PORT || 10500, 10);
```

Comment:
We should create an `env.local` like we do for other services (e.g., https://github.com/Seneca-CDOT/telescope/blob/master/src/api/sso/env.local).    Then pass it into `dev` like this: https://github.com/Seneca-CDOT/telescope/blob/master/src/api/sso/package.json#L7.